video_xblock/static/css/videojs.css

```diff
@@ -220,39 +220,62 @@ body {
     display: none;
 }
 
+.video-js .vjs-custom-caret-button {
+    position: absolute;
+    right: 0px;
+    width: 20px;
+}
+
+.video-js .vjs-custom-transcript-button {
+    position: absolute;
+    right: 20px;
+    width: 40px;
+}
+
+.video-js .vjs-custom-caption-button {
+    position: absolute;
+    right: 61px;
+    width: 40px;
+}
+
 .video-js .vjs-fullscreen-control {
     position: absolute;
-    right: 0;
+    right: 100;
     border-left: 1px dotted #4f595d;
+    border-right: 1px dotted #4f595d;
     width: 41px;
 }
 
 .video-js .vjs-volume-menu-button {
     position: absolute;
-    right: 41px;
+    right: 141px;
     border-left: 1px dotted #4f595d;
     width: 41px;
 }
+
 .video-js .vjs-playback-rate {
     position: absolute;
-    right: 82px;
+    right: 182px;
     border-left: 1px dotted #4f595d;
-    width: 90px;
+    width: 120px;
 }
 
 @media (max-width: 742px) {
-    .vjs-playback-rate-value:before {
+    .vjs-playback-rate-value .label {
         display: none;
     }
     .video-js .vjs-playback-rate {
         width: 41px;
     }
 }
-
 .vjs-playback-rate-value:before {
-    content: 'Speed ';
+    content: '\f0da  Speed ';
```

Comment:
i suggest self.render_resource('../static/css/videojs.css', **context)
in css this can do as "content: attr(data-attr);"